### DISCUSSION TITLE
Quest language is not localized.

### DISCUSSION THREAD

**Discussion by finaln1** [azerothcore/azerothcore-wotlk#16774] [General]
hello.

I wanted to play version 3.3.5 in Korean, so I did the following.

1. Add koKR data to quest_offer_reward_locale, quest_request_items_locale, quest_template_locale tables
2. Set RealmZone = 6 (Korea) in worldserver.conf
3. Set DBC.Locale = 1 (Korea) in worldserver.conf
4. Set SET locale "koKR" in Korean client Config.wtf

However, quests are still displayed in English.

Any workaround? help me please.
thank you

**Comment by vipwuzixu** [comment]
i met the same problem.
